tests/Fixtures/TestBundle/Entity/FourthLevel.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
+
+/**
+ * Fourth Level.
+ *
+ * @author Alan Poulain <contact@alanpoulain.eu>
+ *
+ * @ApiResource
+ * @ORM\Entity
+ */
+class FourthLevel
+{
+    /**
+     * @var int The id
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
```

Comment:
not needed

Reply:
This is needed because of https://github.com/alanpoulain/core/blob/3ef7c9e556e03552597db393d5c52adc59675053/features/main/subresource.feature#L54

Reply:
Thought `strategy="AUTO"` was the default value.

Reply:
Oh you meant that, OK!